01_intro.ipynb

```diff
@@ -1492,15 +1492,15 @@
     "    label_func=is_cat, item_tfms=Resize(224))\n",
     "```\n",
     "\n",
-    "The third line tells fastai what kind of dataset we have, and how it is structured. There are various different classes for different kinds of deep learning dataset and problem--here we're using `ImageDataLoaders`. The first part of the class name will generally be the type of data you have, such as image, or text. The second part will generally be the type of problem you are solving, such as classification, or regression.\n",
```

Comment:
I realize this could be and probably was in a lambda - but as is I think matching a line being a line will help the targeted reader